server/sonar-server/src/main/java/org/sonar/server/rule/ws/UpdateAction.java

```diff
@@ -158,7 +160,8 @@ public void define(WebService.NewController controller) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    checkPermission();
+    ruleWsSupport.checkQProfileAdminPermission();
+
     DbSession dbSession = dbClient.openSession(false);
```

Comment:
at that point of time I had the feeling that doing a mass fix would be more relevant :-)